api/pkg/cli/cmd/info/info.go

```diff
@@ -139,8 +139,68 @@ func (opts *options) run() error {
 	}
 
 	hubClient := opts.cli.Hub()
+
+	// Get all Catalogs
+	catalog, err := hubClient.GetCatalogsList()
+	if err != nil {
+		return err
+	}
+
+	selectOption := so.Options{}
+
+	if len(catalog) == 1 {
+		opts.from = catalog[0]
+	} else if opts.from == "" {
+		// Ask the catalog name
+		err = selectOption.Ask("catalog", catalog)
+		if err != nil {
+			return err
+		}
+		opts.from = selectOption.Catalog
+	}
+
+	// Get all resources from the Catalog selected
+	resources, err := hubClient.GetResourcesList(hub.SearchOption{
```

Comment:
this can be done only when user hasn't passed resource name or catalog

Reply:
if user has passed resource name then this call can be avoided

Reply:
Yeah that makes sense 👍🏻